### ISSUE TITLE
Fix: Make the site responsive for devices of width <720px

### ISSUE THREAD

**Issue by jessej-samuel** [jessej-samuel/spotipy#6]
Currently, the app doesn't support devices with screen-width below, 720px. You can switch the display [here](https://github.com/jessej-samuel/spotipy/blob/8dc95606b6a17477496af7ee663f1bf96306dff4/src/index.css#L40) for seeing the issue.

**Comment by YehorKholodiuk** [comment]
May be you should try to target to  some specific range , while combining Max-width and min-width ?

**Comment by jessej-samuel** [comment]
> May be you should try to target some specific range , while combining Max-width and min-width ?

Yeah. But, I need to rewrite the whole styling part for that. Especially, the player section. Unfortunately, I am quite busy with academics atm 😅

**Comment by jessej-samuel** [comment]
Also, for some reason, I can't seem to place the volume button properly on a small screen.

**Comment by asamaiya00** [comment]
Hi @jessej-samuel, may I work on this issue? Please assign it to me

**Comment by jessej-samuel** [comment]
Yep sure!

**Comment by asamaiya00** [comment]
It needs refactoring, the CSS display property is used in both flex as well as the grid system. Feels a little inconsistent and each component is styled differently. I might not have the bandwidth to refactor all the styles files. 

Unassigned myself. Anyone else can take this up

**Comment by jessej-samuel** [comment]
> It needs refactoring, the CSS display property is used in both flex as well as the grid system. Feels a little inconsistent and each component is styled differently. I might not have the bandwidth to refactor all the styles files. 
> 
> Unassigned myself. Anyone else can take this up

Yea, thanks for your time. I still regret the bad code 🥲
